src/components/Shared/Atoms/Title.tsx

```diff
@@ -5,6 +5,7 @@ export const TitleBig = styled.h2`
   text-align: center;
   ${font_settings(3, 'ita', 500)};
   color: ${({ theme }) => theme.purpleDark};
+  margin: 5px auto;
```

Comment:
takich styli lepiej nie dodawać do Atomu, bo później będziesz chciałą gdzieś to jeszcze wykorzystać i ten margin może przeszkadzać. Lepiej w docelowym miejscu zrobić coś w stylu: 

```
const StyledTitle = styled(Title)`
  margin: 5px auto;
`
```

Reply:
Z założenia ma być to tytuł, który zawsze jest  w jakimś kontenerze i powinien mieć identyczne zachowanie. 
Używamy tego komponentu w kilku miejscach i ma dokładnie identyczny styl wszędzie. 
Żeby to zmienić, trzeba w kilku miejscach teraz napisać taki sam kawałek kodu: 

const StyledTitle = styled(Title)`
  margin: 5px auto;
`

Reply:
ok, skoro tak to może zostać